learning/tour-of-beam/frontend/lib/pages/welcome/screen.dart

```diff
@@ -0,0 +1,348 @@
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(
+        vertical: TobSizes.size20,
+        horizontal: 27,
+      ),
+      child: Column(
+        children: _modules
+            .map(
+              (module) => _Module(
+                title: module,
+                isLast: module == _modules.last,
+              ),
+            )
+            .toList(),
+      ),
+    );
+  }
+
+  static const List<String> _modules = [
+    'Core Transforms',
+    'Common Transforms',
+    'IO',
+    'Windowing',
+    'Triggers',
+  ];
+}
+
+class _SdkSelectionBody extends StatelessWidget {
+  final Widget child;
+  const _SdkSelectionBody({required this.child});
+
+  @override
+  Widget build(BuildContext context) {
+    return DecoratedBox(
+      decoration: BoxDecoration(
+        color: ThemeColors.of(context).background,
+        border: Border(
+          right: BorderSide(
+            color: ThemeColors.of(context).divider,
+          ),
+        ),
+      ),
+      child: child,
+    );
+  }
+}
+
+class _IntroText extends StatelessWidget {
+  const _IntroText();
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        Text(
+          'pages.home.title',
+          style: Theme.of(context).textTheme.displayMedium,
+        ).tr(),
+        Container(
+          margin: const EdgeInsets.symmetric(vertical: 32),
+          height: 2,
+          color: TobColors.grey2,
+          constraints: const BoxConstraints(maxWidth: 150),
+        ),
+        RichText(
+          text: TextSpan(
+            style: Theme.of(context).textTheme.bodyLarge,
+            children: [
+              TextSpan(
+                text: 'pages.home.ifSaveProgress'.tr(),
+              ),
+              TextSpan(
+                text: 'pages.home.signIn'.tr(),
+                style: Theme.of(context)
+                    .textTheme
+                    .bodyLarge!
+                    .copyWith(color: ThemeColors.of(context).primary),
+                recognizer: TapGestureRecognizer()
+                  ..onTap = () {
+                    // TODO(nausharipov): sign in
+                  },
+              ),
+              TextSpan(text: '\n\n${'pages.home.selectLanguage'.tr()}'),
+            ],
+          ),
+        ),
+      ],
+    );
+  }
+}
+
+class _Buttons extends StatelessWidget {
+  const _Buttons();
+
+  void _onSdkChanged(String value) {
+    // TODO(nausharipov): select the language
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Wrap(
+      children: [
+        Wrap(
+          children: ['Java', 'Python', 'Go']
+              .map(
+                (e) => _SdkButton(
+                  value: e,
+                  groupValue: _sdk,
+                  onChanged: _onSdkChanged,
+                ),
+              )
+              .toList(),
+        ),
+        ElevatedButton(
```

Comment:
On a wide screen, for me the "Start learning" button appeared on the right side of the "Java", "Python", and "Go" buttons.  Yet, when I made the width more narrow, the "Start learning" button positioned below.

May we persist the "Start learning" below the language selection buttons?

Reply:
When "Start leaning" is below, it is off a module grid and is not aligned to anything, i.e. it is wider than Java and narrower than Java + Python. It is not that much of a problem, but one line looks better. So I prefer to break only when we have to.

Compare:

![image](https://user-images.githubusercontent.com/44893228/186629445-fa8fffae-4ecc-4805-8a0b-ad4cebadec51.png)

![image](https://user-images.githubusercontent.com/44893228/186628405-fc73de04-bdb0-48bf-8463-e3cf8b17af4b.png)

Also I don't think many people notice the switch as they do not normally resize splash screens like this.